README.md

```diff
@@ -199,6 +199,38 @@ pipeline.fit(X_train, Y_train)
 predictions = pipeline.predict(X_test)
 ```
 
+## Explaining Hierarchical Classifiers
+Hierarchical classifiers can provide additional insights when combined with explainability methods such as SHAP values. Below is a simple example to demonstrate how to calculate hierarchical SHAP values:
+```python
+from hiclass import LocalClassifierPerParentNode, Explainer
+from sklearn.ensemble import RandomForestClassifier
+import numpy as np
+
+# Define data
+X_train = np.array([[1], [2], [3], [4]])
+X_test = np.array([[4], [3], [2], [1]])
+Y_train = np.array([
+    ['Animal', 'Mammal', 'Sheep'],
+    ['Animal', 'Mammal', 'Cow'],
+    ['Animal', 'Reptile', 'Snake'],
+    ['Animal', 'Reptile', 'Lizard'],
+])
+
+# Use random forest classifiers for every node
+rf = RandomForestClassifier()
+classifier = LocalClassifierPerParentNode(local_classifier=rf, replace_classifiers=False)
+
+# Train local classifier per node
+classifier.fit(X_train, Y_train)
+
+# Predict
+predictions = classifier.predict(X_test)
+
+# Explain
+explainer = Explainer(classifier, data=X_train, mode="tree")
+explanations = explainer.explain(X_test)
+```
+
```

Comment:
I don't think this belongs in the quick start in the README. Please, note that I only put a simple example here in order not to overwhelm the reader with information. All other features I added an index in the features section with a hyperlink to their respective examples in the gallery of examples. You can also put hyperlinks to the google colabs notebooks there.

Reply:
The README has been updated - 
- Added Hierarchical Explainability in the features subsection.
- **Removed** code example from Quick start.
- Added some text with links to colab notebook (LCPPN only with this PR) and link to Hierarchical Explainability page on Read the Docs.

Marking resolved (by https://github.com/scikit-learn-contrib/hiclass/pull/106/commits/ae48bf02a40d4cfcac8aae4586b1107d529167e3).